Displays/PIXIE.txt

```diff
@@ -0,0 +1,117 @@
+PIXIE
+
+
+DCPU-16 Hardware Info:
+    Name: PIXIE - Compatible Video Generator
+    ID: 0x774df615, version: 0x1802
+    Manufacturer: 0x00000000 (Manufacturer not yet found)
+
+
+Description:
+    The PIXIE is a 128x96 pixel color display fully compatible with the
+    famous Nya Elektriska's LEM1802. PIXIE is a perfect way to upgrade
+    your DCPU and start enjoying the latest games and graphics.
+
+    PIXIE has the same interrupt behaviour as LEM1802, but it also supports
```

Comment:
behaviour == behavior... second is american, first is everywhere else.  Let's make the spec consistent!  If "AMD"(lol) is going to be "british-y" or whatever, change all the variants, like color => colour  (Note, not saying consistent with the rest of the specs, but if a british company was releasing specs, all their specs would likely be consistent with each other)  If you aren't going for that "british-y" kind of thing, just switch it to behavior.